README.md

```diff
@@ -353,6 +353,17 @@ which you can provide as `baseIRI: string` to the config.
 | dT             | ✓ | ✓ | ? |   |   |
 | bool           | ✓ | ✓ | ? |   |   |
 
+### SPARQL 1.2
+
+Sparqlee looks forward and already implements some SPARQL 1.2 specification functions.
+
+Currently, this is restricted to the [extended date](https://github.com/w3c/sparql-12/blob/main/SEP/SEP-0002/sep-0002.md) functionality.
+Please note that the new build-in `ADJUST` function has not been implemented due to package dependencies.
+
+For the extended date functionality, an additional context item has been added, `implicitTimezone`.
```

Comment:
Can we move this paragraph to the config section of the README? (and add `implicitTimezone` in the snippet there)

Reply:
Yep